addons/test_discuss_full/tests/test_avatar_card_tour.py

```diff
@@ -49,6 +49,7 @@ def setUpClass(cls):
                 "job_id": job.id,
                 "address_id": other_partner.id,
                 "work_email": "test_employee@test.com",
+                "work_phone": "987654321",
```

Comment:
If the values are no longer sync in this case (I guess it is why it had to be added manually?) then I'd suggest a different value than the one of the partner, so we can actually see what difference does it make and which one is used in the test.